### ISSUE TITLE
No logs under X/XWayland

### ISSUE THREAD

**Issue by exalented** [lutris/lutris#5676]
### Bug description

Occasionally, the log window will reset, but not show any logs for any app until lutris is restarted.

### How to Reproduce

 This can be induced by:
1. **Launch**ing lutris under X/Xwayland
2. **'Configure'** a game.
3. **Quickly** launch the game.
4. See the log window reset **logs** and **show blank**.

This issue does not occur when `DISPLAY` is unset and the app runs only in wayland. This happens with an `EVGA GTX 1080 FTWv1` as well.

### Expected behavior

The logs always show app logs.

### Log output

```shell
Log output is no different.
```

### System Information

```shell
lutris 0.5.17-7

Vulkan support: YES
Esync support: YES
Fsync support: YES
Wine installed: YES
Gamescope: YES
Mangohud: YES
Gamemode: NO
Steam: YES
In Flatpak: NO
[System]
OS: Arch Linux rolling n/a
Arch: x86_64
Kernel: 6.11.4-arch2-1
Desktop: sway
Display Server: wayland
[CPU]
Vendor: GenuineIntel
Model: Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz
Physical cores: 4
Logical cores: 8
[Memory]
RAM: 31.3 GB
Swap: 0.0 GB
[Graphics]
Vendor: Intel
OpenGL Renderer: Mesa Intel(R) Arc(tm) A750 Graphics (DG2)
OpenGL Version: 4.6 (Compatibility Profile) Mesa 24.2.5-arch1.1
OpenGL Core: 4.6 (Core Profile) Mesa 24.2.5-arch1.1
OpenGL ES: OpenGL ES 3.2 Mesa 24.2.5-arch1.1
Vulkan Version: 1.3.295
Vulkan Drivers: Intel(R) Arc(tm) A750 Graphics (DG2) (1.3.289)
```

**Comment by danieljohnson2** [comment]
This is very odd, and I was not able to repro it; possibly I don't click fast enough.

If I may ask- does closing and re-opening the log window help? Or doing that, and then starting that game? Or maybe starting a *different* game (they have separate log buffers you see)?

**Comment by exalented** [comment]
> very odd

Yes. This has been happening for quite some time (couple years actually). I haven't submitted an issue because it's quite hard to track what does/does not trigger it.
> closing and re-opening log window help?

No
I should mention that once this happens and you start a different game + open the log buffer for that game it happens for each buffer until lutris is closed.

**Comment by danieljohnson2** [comment]
This suggests it's not the log buffer that is  broken- each game gets a new one. Perhaps the CSS?

If you can repro, perhaps you can provide a screen shot of the busted log window? Maybe there will be a clue.

**Comment by exalented** [comment]
![image](https://github.com/user-attachments/assets/4b05e9cc-b965-4bb8-a640-82da45c15ef4)
This is what it looks like after the game has started and is running.

**Comment by strycore** [comment]
Xorg is no longer supported. Use Wayland.

**Comment by exalented** [comment]
Is there a configuration file where I can disable it instead of using the `DISPLAY` environment variable?

**Comment by danieljohnson2** [comment]
Could you set ``DISPLAY`` in the Lutris game, runner or global options (that's in Preferences)? Would that get the games to work in XWayland when Lutris does not?

The environment variables defined there are set for the games Lutris starts, but not Lutris itself. Best of both worlds, if it works.

**Comment by exalented** [comment]
> Could you set `DISPLAY` in the Lutris game, runner or global options (that's in Preferences)? Would that get the games to work in XWayland when Lutris does not?
> 
> The environment variables defined there are set for the games Lutris starts, but not Lutris itself. Best of both worlds, if it works.

I have tried this. It does not work because the log window does as described above. I run Sway/XWayland.

**Comment by danieljohnson2** [comment]
Hmm. Though the bug only occurs if you run Lutris under XWayland (or real X11); but apparently it also happens if you run only the *a game* under XWayland?

That is super strange, if I have understood you properly.

Does it also happen if you run Lutris and a game under Wayland only, but run some other unrelated XWayland program?

**Comment by exalented** [comment]
> This is super strange

_I should mention I used yofi up until this point. I have now switched to walker and haven't experienced this bug yet with it or when launching with kitty._

This bug DOES NOT occur when Lutris is running on Wayland; though, wine needs an X `DISPLAY` temporarily to easily configure a bottle for wine Wayland only.

When `DISPLAY` is properly set for when Lutris launches under Sway/XWayland this bug seems to occur somewhat randomly with no logs I can detect.

**Comment by legluondunet** [comment]
can you still reproduce this issue with latest Lutris Github code?
Without answer, this ticket will be close in a week.
